README.md

```diff
@@ -1,10 +1,11 @@
+| <sub>[MESD]</sub>                                                                                 | <sub>2022</sub> | <sub>864 audio files of single-word emotional utterances with Mexican cultural shaping.</sub>                                                         | <sub>6 emotions provides single-word utterances for anger, disgust, fear, happiness, neutral, and sadness.</sub>                                                                                                                                                             | <sub>Audio</sub>              | <sub>0,097 GB</sub>    | <sub>Spanish (Mexican)</sub>                                      | <sub>[The Mexican Emotional Speech Database (MESD): elaboration and assessment based on machine learning]</sub>                                                              | <sub>Open</sub>                | <sub>[CC BY 4.0]</sub>       
+|<sub>[MLEnd]</sub>                                                                                 | <sub>2021</sub> | <sub>~32700 audio recordings files produced by 154 speakers. Each audio recording corresponds to one English numeral (from "zero" to "billion")</sub> | <sub> Intonations: neutral, bored, excited and question.</sub>                                                                                                                                                                                                               | <sub>Audio</sub>              | <sub>2.27 GB</sub>     | <sub>--</sub>                                                  | <sub>--</sub>                                                                                                                                                                | <sub>Open</sub>                | <sub>Unknown</sub>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
If possible can you please add the languages list (alphabetical order), I know it is a lot but it will help people navigate the list much easier.

Reply:
(This is optional) ;)